js/main.js

```diff
@@ -1,2 +1,139 @@
 $(function () {
+
+  //let randomChoice = generateRandomChoice()
+  let computerChoice
+  let userChoice
+
+  //generate random choice for the computer
+
+  function generateRandomChoice() {
+    randomChoice = Math.floor(Math.random() * 3)
+
+  if (randomChoice === 0) {
+    //console.log('rock')
+    computerChoice = 'rock'
+  } else if (randomChoice === 1) {
+    computerChoice = 'paper'
+  } else if (randomChoice === 2) {
+    //console.log('scissors')
+    computerChoice = 'scissors'
+  } else {
+    return'something is wrong'
+  }
+}
+  //console.log(computerChoice)
+
+
+  //make rock button an active link
+  $('#rock').click(() => {
```

Comment:
@anniecdavis27 nice work here, great job using the `generateRandomChoice()` and `compareChoices()` function to extract resuable logic